doc/examples/features_detection/plot_blob.py

```diff
@@ -36,13 +36,14 @@
 detected accurately. See :py:meth:`skimage.feature.blob_doh` for usage.
 """
 
+from math import sqrt
 from skimage import data
 from skimage.feature import blob_dog, blob_log, blob_doh
-from math import sqrt
 from skimage.color import rgb2gray
 
 import matplotlib.pyplot as plt
```

Comment:
Why moving math on the top and not this one? if we do external and then internal...